client/components/deposits-overview/index.tsx

```diff
@@ -133,7 +138,7 @@ const DepositsOverview: React.FC = () => {
 				) }
 			</CardBody>
 
-			{ showRecentDeposits && (
+			{ hasRecentDeposits && (
```

Comment:
Nice – only show deposits history & heading if there are some deposits to show 👌🏻